x/executionlayer/client/rest/rest.go

```diff
@@ -17,6 +19,9 @@ func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, storeName string)
 	r.HandleFunc(fmt.Sprintf("/%s/bond", storeName), bondHandler(cliCtx)).Methods("POST")
 	r.HandleFunc(fmt.Sprintf("/%s/unbond", storeName), unbondHandler(cliCtx)).Methods("POST")
 	r.HandleFunc(fmt.Sprintf("/%s/balance", storeName), getBalanceHandler(cliCtx, storeName)).Methods("GET")
+	r.HandleFunc(fmt.Sprintf("/%s/validator", storeName), getValidatorHandler(cliCtx, storeName)).Methods("GET")
+	r.HandleFunc(fmt.Sprintf("/%s/validator", storeName), createValidatorHandler(cliCtx)).Methods("POST")
+	r.HandleFunc(fmt.Sprintf("/%s/validator", storeName), editValidatorHandler(cliCtx)).Methods("PUT")
```

Comment:
why they are under the `contract`? and it seems to be better to use plural form. eg: `validators`

Reply:
1. I put it in the same location as transfer, bond, unbond, and I'm going to move the whole part below hdac after the pull req.
2. The basic operation is for one validator. And I think the validators are right for the whole query, but I'm not sure it's better than creating a new router. Please comment. @hdac-io/developers

Reply:
Better to be in `hdac`, but not now. Prefer to next sprint altogether

Reply:
IMHO, a command which queries all validators would be a plural rather than being a singular.

Reply:
Hmm.. So you guys mean that creating multiple validators is unusual, so you made this API get a single validator. right?

Reply:
https://restfulapi.net/rest-api-design-tutorial-with-example/
all the rest api design said using plural. `validators` +1

Reply:
Then, I'll obey to standard, too

Reply:
https://github.com/hdac-io/friday/pull/65/commits/9149690ece5cff12cd72be088e9ba35ee08a315f